### DISCUSSION TITLE
Player Rater

### DISCUSSION THREAD

**Discussion by nkitaw** [cwendt94/espn-api#474] [Ideas] [answered]
Hey not seeing anyway to currently pull in player rater for basketball api. Would be super helpful have thanks!

**Comment by cwendt94** [comment]
Yes that would be a good stat to have for the player! We have it for football called [posRank](https://github.com/cwendt94/espn-api/wiki/Player-Class) which is the positional ranking of the player. I can add that to basketball as well

**Reply by cwendt94** [reply, marked as answer]
posRank have been added to the [player class](https://github.com/cwendt94/espn-api/wiki/Player-Class-Basketball) in the new version of the package v0.33.0
